.shopware-extension.yml

```diff
@@ -1,25 +1,27 @@
 build:
-    shopwareVersionConstraint: "^6.5.4"
+    shopwareVersionConstraint: "~6.5.4"
 
     zip:
-        composer:
-            before_hooks:
-                - composer install --no-dev
-
         pack:
             excludes:
                 paths:
+                    - .shopware-extension.yml
                     - .gitignore
                     - .github
+                    - .git
                     - bin
-                    - images
+                    - docs
                     - node_modules
+                    - release
                     - tests
+                    - vendor
```

Comment:
I found out that the vendor folder is not needed anymore, after asking a question regarding that on the Community Slack.

The documentation says otherwise, but someone from Shopware told me to ignore that warning for 6.5:
https://developer.shopware.com/docs/guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.html